google-libphonenumber/index.d.ts

```diff
@@ -11,6 +11,21 @@ declare namespace libphonenumber {
         RFC3966
     }
 
+    export enum PhoneNumberType {
+        FIXED_LINE,
+        MOBILE,
+        FIXED_LINE_OR_MOBILE,
+        TOLL_FREE,
+        PREMIUM_RATE,
+        SHARED_COST,
+        VOIP,
+        PERSONAL_NUMBER,
+        PAGER,
+        UAN,
+        VOICEMAIL,
+        UNKNOWN
```

Comment:
https://github.com/googlei18n/libphonenumber/blob/master/javascript/i18n/phonenumbers/phonenumberutil.js#L912-L941

UNKNOWN is -1. 

Should probably enumerate all the enums for compatibility.